frontend/device/kobo/nickel_conf.lua

```diff
@@ -83,6 +88,14 @@ function NickelConf.colorSetting.get()
     return nil
 end
 
+function NickelConf.autoColorEnabled.get()
+    local new_autocolor = NickelConf._read_kobo_conf(re_AutoColorEnabled)
+    if new_autocolor then
+        return (new_autocolor == "true")
+    end
+    return nil
```

Comment:
Seems redundant?

Reply:
If the setting exists, the function should return a boolean, and 'nil' otherwise. I was under the impression that 'nil' and 'false' are different things in Lua?

Reply:
nil ~= false, but the default return value of a function is nil, not false, so that distinction isn't relevant here. ;-) Any return nil at the end of a function is redundant. (Just like in the one a few lines above. I either missed that or decided not to nitpick it.)

Reply:
Thank you for your patience, now I get it. I'm learning Lua as I stumble along... :-)